adminSiteClient/Forms.tsx

```diff
@@ -192,14 +192,29 @@ export class TextAreaField extends React.Component<TextFieldProps> {
                         )}
                     </label>
                 )}
-                <textarea
-                    className="form-control"
-                    value={props.value}
-                    onChange={this.onChange}
-                    onBlur={this.onBlur}
-                    rows={5}
-                    {...passthroughProps}
-                />
+                <div className="input-group">
+                    <textarea
+                        className="form-control custom-control"
```

Comment:
Any particular reason you added the `custom-control` class? I'm asking because it adds a little too much padding on the left and puts the text area in the foreground such that the button's left border is hidden.

Reply:
Good point, this might well be unnecessary. I was looking through the bootstrap docs and they didn't have an example of a button with a textarea and the example use that I found had the custom-control class which is probably redundant. Will see if we can remove this.